README.md

```diff
@@ -1,239 +1,23 @@
 
+This plug-in can also be used as a sample to build your own custom credential manager to use based on your own use case.
 
 ## How the plug-in works
 
-The plug-in invokes a native Node module that manages user IDs and passwords in the default credential manager in your computer's operating system.
-
-When you install Zowe CLI but do not install the plug-in, your Zowe CLI profiles display as plain text in your default credential manager. For example:
-
-```
-type: zosmf
-host: test
-port: 1234
-user: USERNAME
-pass: PASSWORD
-rejectUnauthorized: false
-```
-
-With the plug-in installed and configured, the plug-in lets you store your Zowe CLI credentials securely. For example:
-
-```
-type: zosmf
-host: test
-port: 1234
-user: 'managed by @zowe/secure-credential-store-for-zowe-cli'
-pass: 'managed by @zowe/secure-credential-store-for-zowe-cli'
-rejectUnauthorized: false
-```
-<!-- TODO: Verify that @zowe/secure-credential-store is correct -->
-
-The initial behavior of the plug-in varies based on the method that you use to install the plug-in.
-
--   When you install the plug-in using the *Windows Installation Wizard*, the installation process starts the plug-in and updates all of your existing Zowe CLI profiles automatically. No further action on your part is required.
--   When you install the plug-in from *an online registry or a local package*, the plug-in starts, however, you must update your existing Zowe CLI profiles manually.
-
-## Software requirements
-
-Before you install and use the plug-in:
-
--   Install Zowe CLI on your computer.
-    For more information, see [Installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
--   NPM version 7 (npm@7) and NPM version 8 (npm@8) requirements:
-    On Windows operating systems, ensure that your computer is connected to the Internet and issue the following command ***before*** you install the plug-in:
-    ```
-    npm install -g prebuild-install
-    ```
-### Linux (graphical)
-
--   Install the packages `gnome-keyring` and `libsecret` (or `libsecret-1-0` on Debian and Ubuntu).
-
-### Linux (headless)
-
--   Install the packages listed for graphical Linux above.
-
--   Run the commands below to unlock the Gnome keyring. The keyring must be unlocked again for each new user session.
+The plug-in uses the Kubernetes API in order to communicate with your Kubernetes cluster and store secrets on a per user basis.
 
-    The second command will prompt for your password. Press Ctrl+D twice when you have finished typing it.
-    ```bash
-    export $(dbus-launch)
-    gnome-keyring-daemon -r --unlock --components=secrets
-    ```
+In addition, when storing Kubernetes secrets, the secret will be stored in the assigned namespace for the current context or otherwise through an environment variable named `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`.
```

Comment:
```suggestion
In addition, when storing Kubernetes secrets, users can specify a namespace through the environment variables `$DEVWORKSPACE_NAMESPACE` and `$WORKSPACE_NAMESPACE`. Otherwise, the secret will be stored in the assigned namespace for the current context.
```